docs/sphinx/source/whatsnew/v0.7.2.rst

```diff
@@ -10,6 +10,12 @@ API Changes
 * :py:func:`pvlib.iotools.read_tmy3` when coerced to a single year now returns
   indices that are monotonically increasing. Therefore, the last index will be
   January 1, 00:00 of the *next* year. (:pull:`910`)
+* Renamed :py:mod:`pvlib.losses` to :py:mod:`pvlib.soiling`. Additional loss
```

Comment:
I believe in this version of the docs, there is no module called `pvlib.losses`, so maybe change this to 

    * Renamed ``pvlib.losses`` to :py:mod:`pvlib.soiling`. Additional loss